app/controllers/admin/members_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Admin::MembersController < Admin::ApplicationController
+  def index
+    @members = Admin.all
+  end
+
+  def new
+    @member = Admin.new
+  end
+
+  def create
+    @member = Admin.new(admin_params)
+
+    if @member.save
+      flash[:notice] = t('.success')
+      redirect_to admin_members_path
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def create_password
+    Devise.friendly_token.first(8)
+  end
+
+  def admin_params
+    with_password = params.require(:member).permit(:email, :name)
+    with_password[:password] = create_password
+    with_password
```

Comment:
```suggestion
    { **with_password, password: Devise.friendly_token.first(8) }
```

Se vocês trocarem essa linha pra isso vocês evitam até o método da senha de existir 😄

Reply:
Mas eu particularmente acharia legal esse método ser só as coisas normais e lá onde você precisa você faz a customização, 

```ruby
def admin_params
  params.require(:member).permit(:email, :name)
end
```

```ruby
@member = Admin.new(**admin_params, password: Devise.friendly_token.first(8))
```

Porque aí fica bem separado o que vem do form dos parâmetros novos que você quer colocar. O operador `**` chama double splat operador ele pode ser usado pra juntar dois hashes

Reply:
Sim!
Fizemos tudo isso porque não estávamos conseguindo lembrar do nome "double splat", acabamos de implementar estas sugestões e já revisamos as interferências com outros commits.
acabamos de enviar as alterações neste novo commit.